src/commonmark/en/content/user/importexport-app.md

```diff
@@ -149,8 +175,25 @@ import.
 
 4.  Choose **Org Unit ID Scheme**.
 
-5.  Click **Import** which will bring up the **Logger** at the bottom of
-    the page to show the progress.
+1.  Click on the **Import** button which will upload the file and start the
+    importing process.
+
+### GML Import
+
+<!--DHIS2-SECTION-ID:gml_import-->
+
+GML Import can be accessed from the sidebar by clicking on GML
+Import.
+
+![](resources/images/import_export/gml_import.png)
+
+1.  Upload a file using the *GML* (Geographic Markup Language) format.
+
+2.  Choose **Dry Run** . *Yes* will do a test import without importing
+    any data into the database.
+
+1.  Click on the **Import** button which will upload the file and start the
+    importing process.
 
 ## Metadata Import Summaries
```

Comment:
Would it make sense for this now to be a h3 level?
Should it also be simply "Import Summaries"?
(maybe it also belongs next to the "Import process logger" section?)

Reply:
Oops, yes, that should be an h3 as well.
I'll move it below the "Import process logger" section as you've suggested

Reply:
Done